layouts/index.html

```diff
@@ -17,18 +17,59 @@
 
 <script type="text/javascript">
   window.onload = function() {
+    // form in hero
     var button = document.getElementById("mc-embedded-subscribe")
 
     button.onclick = function (event) {
+      event.preventDefault()
 
-      if(document.getElementsByClassName("cta-email")[0]) {
+      if(document.getElementsByClassName("cta-email") && document.getElementsByClassName("cta-email")[0]) {
 
-        var inputValue = document.getElementsByClassName("cta-email")[0].value
+        var inputValue = document.getElementsByClassName("cta-email")[0].value;
```

Comment:
Why not move these two lines to be above the if statement and use the declared variables in the if statement instead.